src/sindy/isindy.jl

```diff
@@ -30,35 +30,35 @@ function ISInDy(X::AbstractArray, Ẋ::AbstractArray, Ψ::Basis; alg::Optimize.A
     # Init for sweep over the differential variables
     Ξ = zeros(eltype(θ), length(Ψ)*2, size(Ẋ, 1))
 
-    opt_front = ParetoFront(1, scalarization = alg)
-    tmp_front = ParetoFront(1, scalarization = alg)
 
+    fg(xi, theta) = (g∘f)(xi, theta)
+
+    iters = 0
 
-    @inbounds for i in 1:size(Ẋ, 1)
+    @simd for i in 1:size(Ẋ, 1)
```

Comment:
this isn't a simd-able loop. The macro will just be a no-op

Reply:
Alright. I will change that.